lix/source-code/client/package.json

```diff
@@ -27,9 +27,16 @@
 	"dependencies": {
 		"@lix-js/fs": "workspace:*",
 		"@octokit/types": "12.4.0 ",
-		"isomorphic-git": "1.24.5",
 		"octokit": "3.1.2",
-		"solid-js": "1.7.11"
+		"solid-js": "1.7.11",
+		"async-lock": "^1.4.1",
```

Comment:
Do we need those packages after we have vendored iso git?

Reply:
@martin-lysk  yes we dont use isogits package and none of their deps, we use pathced versions that have security fixes and only the dependencies that will actually be used in lix